app.js

```diff
@@ -123,11 +128,12 @@ function processStats (data) {
     .then(pushStats)
 }
 
-function getStats () {
+function getStats (shard) {
+  console.log('getting stats (' + shard + ')')
```

Comment:
This is causing duplicated output, since in tick() it outputs `Fetching stats`, Probably better to either add shard to that message or remove it.